lib/rubocop/cop/sorbet/rbi/forbid_rbi_outside_of_allowed_paths.rb

```diff
@@ -21,25 +23,41 @@ class ForbidRBIOutsideOfAllowedPaths < RuboCop::Cop::Cop
         include RangeHelp
 
         def investigate(processed_source)
+          paths = allowed_paths
+
+          if paths.nil?
+            add_offense(
+              nil,
+              location: source_range(processed_source.buffer, 1, 0),
+              message: "AllowedPaths expects an array"
+            )
+            return
+          elsif paths.empty?
+            add_offense(
+              nil,
+              location: source_range(processed_source.buffer, 1, 0),
+              message: "AllowedPaths cannot be empty"
+            )
+            return
+          end
+
+          # When ran with `bundle exec` the path to the source file is absolute.
+          # We need to remove the exec path directory prefix before matching with the filename regular expressions.
+          rel_path = processed_source.file_path.sub("#{Dir.pwd}/", "")
```

Comment:
I'm not super happy about this fix but I don't have a better idea on how to find the relative path to the file from the project root.

My bet is that the paths in the config should match the directory from where you expect to run your rubocop command.

Reply:
> Is the path to the file only absolute when running with `bundle exec`?

I double checked and no, it's actually the same with or without `bundle exec`.

> I'm wondering if it would simplify things to take the `allow_paths` and manipulate them instead.

How would you go about it? Using `"#{Dir.pwd}/#{pattern}"` instead of a bare `pattern`? It seems equivalent.

Reply:
Yeah, I guess so. Another idea is using `Dir` and `Pathname` to get all files that match with the complete path and check if it's included in the list. Something like

```ruby
all_matches = Dir[pattern]
full_paths = all_matches.map { |path| Pathname.new(path).realpath } # full_paths is now a list of all files that match the pattern with the full path
```

But I guess it might be slower and not as elegant as our previous approach.

Reply:
> I double checked and no, it's actually the same with or without bundle exec.

Sad I missed this tbh.
Can you update the comment?